app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -0,0 +1,474 @@
+    color: theme.palette.grey[600],
+    fontWeight: 700,
+    fontSize: theme.typography.button.fontSize,
+    "&&": {
+      borderRadius: theme.shape.borderRadius * 2,
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      border: `2px solid ${theme.palette.primary.main}`,
+    },
+    "&:focus-visible": {
+      border: `2px solid ${theme.palette.primary.main}`,
+      outline: "none",
+      boxShadow: "none",
+    },
+  },
+
+  inputNumberActive: {
+    "&&": {
+      border: `2px solid ${theme.palette.primary.main}`,
+    },
+  },
+}));
+
+export interface DonationFormData {
+  amount: number;
+  recurring: boolean;
+}
+
+export default function DonationsBoxMixed() {
+  const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_KEY!);
+
+  const classes = useStyles();
+
+  const [success, setSuccess] = useState(false);
+  const [cancelled, setCancelled] = useState(false);
+
+  const location = useLocation();
+
+  const [isAmount, setIsAmount] = useState(false);
+
+  useEffect(() => {
+    const query = new URLSearchParams(location.search);
+    setSuccess(query.get("success") ? true : false);
+    setCancelled(query.get("cancelled") ? true : false);
+  }, [location]);
+
+  const {
+    control,
+    handleSubmit,
+    reset: resetForm,
+    errors,
+  } = useForm<DonationFormData>({ defaultValues: { recurring: true } });
+
+  const {
+    error,
+    isLoading,
+    mutate: initiateDonation,
+  } = useMutation<void, GrpcError, DonationFormData>(
+    async (formData) => {
+      const stripe = (await stripePromise)!;
+      const session_id = await service.donations.initiateDonation(formData);
+      // When the customer clicks on the button, redirect them to Checkout.
+      const result = await stripe.redirectToCheckout({
+        sessionId: session_id,
+      });
+      if (result.error) {
+        throw Error(result.error.message);
+      }
+    },
+    {
+      onSuccess: () => {
+        resetForm();
+      },
+    }
+  );
+
+  const onSubmit = handleSubmit((data) => {
+    initiateDonation(data);
+  });
+
+  return (
+    <>
+      <form onSubmit={onSubmit} className={classes.donationsBox}>
+        {error && <Alert severity="error">{error.message}</Alert>}
+        {success && (
+          <Alert severity="success">{DONATIONSBOX_ALERT_SUCCESS}</Alert>
+        )}
+        {cancelled && (
+          <Alert severity="warning">{DONATIONSBOX_ALERT_WARNING}</Alert>
+        )}
+        <Typography className={classes.marginBottom2} variant="h3">
+          {DONATIONSBOX_TITLE}
+        </Typography>
+        <Controller
+          id="recurring"
+          control={control}
+          name="recurring"
+          rules={{
+            validate: (value) =>
+              typeof value === undefined ? DONATIONSBOX_REQUIRED : "",
+          }}
+          defaultValue="monthly"
+          render={({ onChange, value }) => (
+            <FormControl className={classes.formGroup}>
+              <RadioGroup
```

Comment:
There might need to be a `<FormLabel>` here to be tabbable

Reply:
@Jesus82 Did you try this?

Reply:
Adding a `<FormLabel>` above `RadioGroup` didn't help, but I might've done it wrong

Reply:
Solved it. The problem was that the "original" radio button was hidden with `display:none` and that prevented tabbing. Changed it to a fixed position outside the viewport, now tabbing is working fine.